waffle-chai/test/events.ts

```diff
@@ -5,11 +5,14 @@ import {EVENTS_ABI, EVENTS_BYTECODE} from './contracts/Events';
 
 describe('INTEGRATION: Events', () => {
   const [wallet] = new MockProvider().getWallets();
+  let eventFactory: ContractFactory;
   let events: Contract;
 
   beforeEach(async () => {
     const factory = new ContractFactory(EVENTS_ABI, EVENTS_BYTECODE, wallet);
     events = await factory.deploy();
+    eventFactory = new ContractFactory(EVENTS_ABI, EVENTS_BYTECODE, wallet);
+    events = await eventFactory.deploy();
```

Comment:
Why deploy this twice?

Reply:
That must have been a mindless copy-paste or unstashing mistake. Please review again.